src/index.html

```diff
@@ -8,11 +8,47 @@
     />
     <title>Calendar</title>
     <link
+      href="https://fonts.googleapis.com/css2?family=Arial:wght@400;700&display=swap"
       rel="stylesheet"
-      href="styles/index.scss"
+    />
+    <link
+      rel="stylesheet"
+      href="./styles/main.scss"
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-sun calendar--lenght-31">
```

Comment:
There is a typo in the class name `calendar--lenght-31`. It should be `calendar--length-31`.